KBQA/kbqa/docker-compose.yml

```diff
@@ -33,6 +33,15 @@ services:
     expose:
       - 24802
 
+  gerbil:
+    build: ./webservice/gerbil
+    container_name: gerbil
+    restart: always
+    expose:
+      - 24804
+    volumes:
+      - ./gerbil/evaluation:/evaluation
```

Comment:
I don't know, which path is preferred, but we can change the path to "${RESOURCE_PATH}/gerbil/evaluation:/evaluation" just like the other volumes to have it consistent. You can decide on that.

Reply:
Yes, should use the ${RESOURCE_PATH} var